benchmark-runner/analyze.ts

```diff
@@ -2,6 +2,7 @@ import { PackageInfo } from "workspace-tools";
 import { Task } from "./go";
 import fs from "fs-extra";
 import path from "path";
+import { sync as readPkgSync } from 'read-pkg-up';
```

Comment:
It couldn't handle `linking` situations.

Most time, we run benchmarks in CI and the `@speedy/core` is linked to the folder rather than installed.

https://github.com/hardfist/speedystack/blob/17641b31ef3e006ff2d24f56ec7da25fbab9a8a4/.github/workflows/Benchmark.yml#L39-L40

Reply:
I think it's better to use `cp -rf ../../../packages/speedy-core ./benchmarks/node_modules/@speedy-js/speedy-core` to cover origin project.

Reply:
It won't solve your problem. `read-pkg-up` just read the version in `package.json`, rather than the real version of the package installed.

Reply:
🤔️, I think `cp -rf` will solve the problem, because I had passed in the cwd param in `readPkg`, `readPkgSync({cwd: require.resolve(pkgName)})`